source/configure/calls-deployment.rst

```diff
@@ -338,10 +342,37 @@ If you wish to host many calls or calls with a large number of participants, tak
   # Allow to allocate more memory as needed for more control messages that need to be sent for each socket connected
   net.core.optmem_max = 16777216
 
+The rtcd service
+----------------
+
+.. include:: ./calls-rtcd-ent-only.rst
+  :start-after: :nosearch:
+
+The Calls plugin has a built-in `Selective Forwarding Unit (SFU) <https://bloggeek.me/webrtcglossary/sfu/>`__ to route audio and screensharing data. This is the ``integrated`` option described in the `<#modes-of-operation>`__ section above. But this SFU functionality can be deployed separately as an external ``rtcd`` instance.
+
+Reasons to use the ``rtcd`` service
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This section will help you understand when and why your organization would want to use ``rtcd``.
+
+To be clear, ``rtcd`` is a standalone service, which adds operational complexity, maintenance costs, and requires an enterprise licence. For those who are evaluating Calls, and for many small instances of Mattermost, the integrated SFU (the one included in the Calls plugin) may be sufficient.
+
+Having said that, the ``rtcd`` service provides many benefits and is the recommended way to host Calls for the following reasons.
+
+- **Performance and stability of the main Mattermost server(s).** Letting the Calls plugin run the SFU means that calls traffic is being processed by the server that's also processing the rest of the Mattermost data (Channels, Boards, Playbooks, etc). In addition to the raw traffic, as the number of simultaneous screen sharing and voice sessions increase, so too will the CPU processing and memory requirements. If the Calls traffic spikes, it can affect the responsiveness of the rest of the Mattermost services. Using an ``rtcd`` service isolates the calls traffic processing to those ``rtcd`` instances.
```

Comment:
```suggestion
Performance of the main Mattermost server(s). When the Calls plugin runs the SFU, calls traffic is added to the processing load of the server running the rest of your Mattermost services. If Calls traffic spikes, it can negatively affect the responsiveness of these services. Using an rtcd service isolates the calls traffic processing to those rtcd instances, and also reduces costs by minimizing CPU usage spikes.
```